examples/basic/main.tf

```diff
@@ -66,6 +68,8 @@ module "config_us_east_1" {
   config_retention_in_days    = 90
   tags                        = local.tags
   home_region                 = "eu-west-2"
+  config = { default = {
+  rule_groups = {} } }
```

Comment:
Nice try - but given this is an example, any chance we can make one of them a little more than an empty set.

If you scroll down in this file, you'll see a sweet example from line 82. Refactor the code a little and we can reuse the same config in both "module compliance" and "module config_<region>"